src/player/Player.php

```diff
@@ -511,6 +513,14 @@ public function setAutoJump(bool $value) : void{
 		}
 	}
 
+	public function setImmobile(bool $value): void {
+		$this->isImmobile = $value;
+	}
+
+	public function isImmobile(): bool {
+		return $this->isImmobile;
+	}
+
```

Comment:
Why recreate a function `setImmobile` when **Dylan** has just changed the name of the function `setImmobile`, just rename it ?

Reply:
'Entity->setNoClientPredictions()' okay I didn't see lol

Reply:
@Refaltor77 Then you can always change `setNoClientPredictions()` to `setImmobile()`.